src/scripts/components/header/SearchInput.js

```diff
@@ -64,9 +64,7 @@ export default class SearchInput extends React.Component {
       e.target.style.textAlign = 'left';
     }
 
-    if (e.target) {
-      this.setState({ value: e.target.value.trim() });
-    }
+    this.setState({ value: this.refs.search.value.trim() });
```

Comment:
this should be `this.input.value.trim` because `this.refs.search.value` is undefined according to errors in my console (and it breaks suggest)